src/server/api/routes/users/v2/follow.js

```diff
@@ -0,0 +1,91 @@
+const { Router } = require('express');
+const {
+  authenticateV2Login,
+  checkWalletMatch,
+} = require('../../../middleware/auth');
+const { setPrivateCacheHeader } = require('../../../middleware/cache');
+const { handleRestfulError } = require('../../../middleware/error');
+const { isValidAddress } = require('../../../util/cosmos');
+const {
+  FieldValue,
+  walletUserCollection,
+} = require('../../../../modules/firebase');
+
+const router = Router();
+
+router.get('/v2/users/:wallet/followers', async (req, res, next) => {
+  try {
+    const { wallet: user } = req.params;
+    const userDoc = await walletUserCollection.doc(user).get();
+    let followers = [];
+    if (!userDoc.exists) {
+      res.json({ followers });
+      return;
+    }
+    ({ followers } = userDoc.data());
+    res.json({ followers });
```

Comment:
```suggestion
    if (!userDoc.exists) {
      res.json({ followers: [] });
      return;
    }
    const { followers = [] } = userDoc.data();
    res.json({ followers });
```
Seems cleaner this way?